addons/l10n_us/__manifest__.py

```diff
@@ -10,7 +10,8 @@
     'category': 'Accounting/Localizations/Account Charts',
     'description': """
     """,
-    'depends': ['base'],
+    'depends': ['account'],
+    'auto_install': ['account'],
```

Comment:
This is essentially a revert of https://github.com/odoo/odoo/commit/62df6225fed8364990b25428d13ae36b03f4d4a9

If I understand correctly the reason behind this commit, it's that we want to keep `l10n_us` for stuff that must be shared across modules (sales, purchases, pos, ...), and use `l10n_us_account` for accounting related stuff.

So we should either move this to a new module `l10n_us_account` or discuss with the author of the commit.

Reply:
Hello @RicGR98  ! Just to add some context, I created the task (3865230) because clients in the US were having issues installing the localization features. So, I want to understand the solution better :) Creating a new module, isn't risky in stable? Is that the only option?

Thanks!
cc @ren-odoo

Reply:
Hello :)

Yep I understood the problem, however this PR is just reverting another commit (so it was done on purpose but with an unfortunate side effect).

Creating a new module is not risky at all (even in stable), it's actually done extensively at Odoo to fix some things in stable. So there would be `l10n_us` for generic things across POS, Sales, HR, ... and `l10n_us_account` for US Accounting related stuff. You can see that it was the author's purpose here (where Accounting was renamed to generic Localizations):

https://github.com/odoo/odoo/commit/62df6225fed8364990b25428d13ae36b03f4d4a9#diff-2c17c1fb5f69aea850f6095864da4ccf349178993c5e475fff651e8986d9b32aL5

Reply:
@RicGR98 
l10n_us module is actually containing nothing related to accounting. So, There is no actual dependencies on account module but the other packages that depends on account module e.g., l10n_us_reports have an actual dependency on `10n_us`. So, actually there is nothing that can be put in a new module `l10n_us_account`

So, What do you recommend?

Reply:
Hi, @tivisse can you help us with your insight here? Thanks!

Reply:
> So, actually there is nothing that can be put in a new module l10n_us_account

I'm not sure what you mean with this sentence. Why couldn't we create a new module `l10n_us_account` with `'countries': ['us']`,  `'depends': ['l10n_us']`, `auto_install': ['account']` ? Wouldn't that work ?

`l10n_us_account` would be auto installed when the country is US and account is installed. Then `l10n_us` would be installed since it's a dependency of `l10n_us_account`, and then `l10n_us_reports` would also be auto installed since it's dependencies are also met (if you have chosen accounting, not invoicing)

Let me know

Reply:
I thought having kind of empty module would violate something. 
But it seems alright. So, i'll do 👍